packages/consensus/src/codec.cairo

```diff
@@ -407,6 +414,8 @@ mod tests {
                             ),
                             cached: false,
                         },
+                        block_hash: 0x000000000000000931112ca80c5badf6047373b1bb53587fc23344871734bbff_u256
```

Comment:
try `hex_to_hash_rev("000000000000000931112ca80c5badf6047373b1bb53587fc23344871734bbff")`

Reply:
Thanks, that's helped!